fact/coordinates/camera_frame.py

```diff
@@ -32,6 +29,7 @@ class CameraFrame(BaseCoordinateFrame):
     pointing_direction = CoordinateAttribute(frame=AltAz, default=None)
     obstime = TimeAttribute(default=None)
     location = EarthLocationAttribute(default=LOCATION)
+    rotated = Attribute(default=True)
```

Comment:
Should the default be False for the new version?

Reply:
True means we use the new coordinate system, maybe that should be made clear

Reply:
Documented in the docstring of `CameraFrame` now